src/SFA.DAS.RoATPService.Application/Validators/OrganisationValidator.cs

```diff
@@ -14,9 +14,9 @@ public bool IsValidOrganisationId(Guid organisationId)
             return true;
         }
 
-        public bool IsValidApplicationRouteId(int applicationRouteId)
+        public bool IsValidProviderTypeId(int providerTypeId)
         {
-            return (applicationRouteId > 0);
+            return (providerTypeId > 0);
```

Comment:
Is that all the test is?   Is provider type from a lookup?

Reply:
Good point, should range check against the values in the ProviderType table